inc/html.class.php

```diff
@@ -6835,6 +6836,8 @@ private static function displayMainMenu($full, $options = []) {
          }
       }
 
+      $menu = Plugin::doHookFunction("redefine_menus", $menu);
```

Comment:
Needs doc ;)

Reply:
Doc is now ready: https://github.com/glpi-project/docdev/pull/107.